newshomepages/screenshot.py

```diff
@@ -17,11 +15,32 @@
 @click.option("-w", "--wait", "wait", default=5000)
 @click.option("-x", "--width", "width", default=1300)
 @click.option("-y", "--height", "height", default=1600)
-def cli(handle: str, output_dir: str, wait: str, width: str, height: str):
+@click.option(
+    "-f",
+    "--screenshot_full_page",
```

Comment:
The CLI tool is already called `screenshot` so I don't think we need to repeat the name here. I'd also prefer for the input options to use hyphens, not underscores. What do you think of something like `full-page`, which would match the playwright option?